jobrunner/config.py

```diff
@@ -51,6 +55,30 @@
 
 TEMP_DATABASE_NAME = os.environ.get("TEMP_DATABASE_NAME")
 
+PRESTO_TLS_KEY = PRESTO_TLS_CERT = None
+PRESTO_TLS_CERT_PATH = os.environ.get("PRESTO_TLS_CERT_PATH")
+PRESTO_TLS_KEY_PATH = os.environ.get("PRESTO_TLS_KEY_PATH")
+
+if PRESTO_TLS_KEY_PATH and PRESTO_TLS_CERT_PATH:
```

Comment:
If you set one value but not the other (e.g. because you typo'd it) then I think this ends up silently ignoring both values.

I wonder if it would be better to do the two checks independently like:
```py
if PRESTO_TLS_KEY_PATH:
    key_path = Path(PRESTO_TLS_KEY_PATH)
    if key_path.exists():
        PRESTO_TLS_KEY = key_path.read_text()
    else:
        raise ConfigException(
            f"PRESTO_TLS_KEY_PATH={key_path}, but file does not exist"
        )

if PRESTO_TLS_CERT_PATH:
    cert_path = Path(PRESTO_TLS_CERT_PATH)
    if cert_path.exists():
        PRESTO_TLS_CERT = cert_path.read_text()
    else:
        raise ConfigException(
            f"PRESTO_TLS_CERT_PATH={cert_path}, but file does not exist"
        )

if bool(PRESTO_TLS_KEY_PATH) != bool(PRESTO_TLS_CERT_PATH):
    raise ConfigException(
            "Both PRESTO_TLS_KEY_PATH and PRESTO_TLS_CERT_PATH must be defined if either are"
    )
```

Reply:
Good call, done